src/dataviews/dataview-model-base.js

```diff
@@ -225,6 +263,7 @@ module.exports = Model.extend({
     return this.getSourceId() === this.layer.get('source');
   },
 
+  // @return might return undefined if there's no source
```

Comment:
I don't think this is true, given that we're always setting a source in the dataviewFactory.